index.bs

```diff
@@ -1566,12 +1566,33 @@ for each [=sensor types=]:
     especially when doing so would not make sense.
 -   A set of [=identifying parameters=]. TODO: replace that by an abstract operation.
 
-
 <h3 id="permission-api">Extending the Permission API</h3>
 
-Provide guidance on how to extend the Permission API [[PERMISSIONS]]
-for each [=sensor types=].
+An implementation of the {{Sensor}} interface for each [=sensor type=] must protect its
+[=sensor reading|reading=] by associated {{PermissionName}} or {{PermissionDescriptor}}.
+A [=Low-level=] {{Sensor|sensor}} may use its interface name as a {{PermissionName}},
+for instance, "gyroscope" or "accelerometer". [=sensor fusion|Fusion sensors=] must
+[=request permission to use|request permission to access=] each of the sensors that are
+used as a source of fusion.
+
+Even though, it might be difficult to reconstruct [=low-level=] [=sensor readings=] from
+fused data, some of the original information might be inferred. For example, it is easy to
+deduce user's orientation in space if absolute or geomagnetic orientation sensors are used,
+therefore, these sensors must [=request permission to use|request permission to use=]
+magnetometer as it provides information about orientation of device in relation to Earth's
+magnetic field. In contrast, relative orientation sensor does not expose such information, thus,
+it does not need to [=request permission to use|request permission to use=] magnetometer.
+
+{{PermissionDescriptor|Permission descriptors}} can also be used to set maximum allowed limits
+for accuracy or [=sampling frequency=]. An example for a possible extension of the Permission API
```

Comment:
of the PermissionDescriptor interface ?

Reply:
PermissionDescriptor is a dictionary. All API specific dictionaries are defined in Permission API. Actually it would be nice to have something like partial dictionary or namespace in Permission API, so that other specs would not need to modify it whenever new token is required.